File: output_to_file/output_to_file.py
import sys
output_file = "/tmp/output_python.txt"
try:
  fh = open(output_file, "w") or die("Failed to write to " + output_file)
except IOError:
  print("Failed to write to " + output_file)
  sys.exit(1)

fh.write("Hello World!\n")
fh.close()

fh = open(output_file, "r")
print(fh.read(), end="")
fh.close()

File: output_to_file/output_to_file.pl
use strict;
use FileHandle;
my $output_file = "/tmp/output_perl.txt";
if (my $fh = new FileHandle('> '.$output_file)) {
  print $fh "Hello World!\n";
  $fh->close();
}

if (my $fh = new FileHandle($output_file)) {
  local $/ = undef;
  print <$fh>;
  $fh->close();
}

